cfme/infrastructure/provider/rhevm.py

```diff
@@ -71,10 +68,7 @@ def __init__(self, name=None, endpoints=None, zone=None, key=None, hostname=None
     def view_value_mapping(self):
         return {
             'name': self.name,
-            'prov_type': version.pick({version.LOWEST: 'Red Hat Enterprise Virtualization Manager',
-                                       '5.7.1': 'Red Hat Virtualization Manager',
-                                       '5.7.3': 'Red Hat Virtualization',
-                                       '5.8': 'Red Hat Virtualization Manager',
+            'prov_type': version.pick({version.LOWEST: 'Red Hat Virtualization Manager',
```

Comment:
@ohiliazo in all 5.7z, 5.8z & 5.9z, I see `Red Hat Virtualization` in the provider list, my tests related to RHV failed for 5.7 so got here, maybe you can update version from `5.8.0.10` to latest 5.8z
@psav I think this got merged without running PRT, so this got in without failing.